<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphwindow</class>
 <widget class="QDialog" name="Graphwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(145, 52, 162, 92);</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(144, 50, 161, 48);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Функция :</string>
   </property>
  </widget>
  <widget class="QLabel" name="function_is">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>9</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>X min =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="x_min">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_max">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>X max =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="x_max">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Y max =</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgba(37, 37, 37, 117);
    color:  rgb(255, 255, 255);
    font-size:  33px;
	font: 16pt &quot;Comic Sans MS&quot;;
}

QPushButton:pressed {
    background-color:  rgb(148, 59, 165);
}</string>
   </property>
   <property name="text">
    <string>Нарисовать график</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Y min =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_min">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color:  rgb(245, 245, 245);
font: 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
